app/server/tests/api-getCommunity.test.js

```diff
@@ -0,0 +1,136 @@
+require('dotenv').config();
+const supertest = require('supertest');
+
+const endpoint = process.env.REACT_APP_API_REF;
+const request = supertest(endpoint);
+var token = '';
+
+const welcomeCommunityTitle = "Welcome";
+const welcomeCommunityDescript = "Test";
+
+
+describe('Testing the get communities function without logging in', () => {
+
+    test('API returns unsuccessful response', async () => {
+      let response = await request
+       .get('/community')
+
+      expect(response.ok).toBe(false);
+    });
+
+    test('API returns code 401', async () => {
+      let response = await request
+        .get('/community')
+
+      expect(response.status).toBe(401);
+    });
+
+    test('API returns description "Not Authorized."', async () => {
+      let response = await request
+        .get('/community')
+
+      expect(response.text).toBe("Not Authorized.");
+    });
+});
+
+
+
+describe('Testing login for test user', () => {
+
+  test('Test account can log in', async () => {
+      let response = await request
+        .post('/login')
+        .send({
+            "name": "test",
+            "password": "Test123!"
+        });
+
+      token = response.body.token;
+      expect(response.status).toBe(201);
+  });
+});
+
+
+
+describe('Testing the get communities function after logging in', () => {
+
+  test('API returns successful response', async () => {
```

Comment:
I feel like the test that starts on 57 and 65 to be very similar. Can you explain what the difference is?

Reply:
like incase it gave a 201, it would get caught there.